.drone.yml

```diff
@@ -85,6 +85,17 @@ pipeline:
     when:
       event: [ push, tag ]
 
+  make-rpm:
```

Comment:
<bikeshed>package-rpm for consistency?</bikeshed>

Reply:
✅